tests/bot/test_converters.py

```diff
@@ -16,7 +16,7 @@
 )
 
 
-class ConverterTests(unittest.TestCase):
+class ConverterTests(unittest.IsolatedAsyncioTestCase):
```

Comment:
Good idea w/ using `IsolatedAsyncioTestCase`, I was actually just revisiting the PR and about to suggest using it instead of `asyncio.run()`. `asyncio.run()` has a fair bit of additional overhead that isn't needed for testing purposes, especially when you call it multiple times (it creates and closes a new event loop each time it's used, in addition to some other cleanup calls related to tasks, async generators, and in 3.9 the default executor).